website/docs/d/instance.html.markdown

```diff
@@ -69,6 +69,7 @@ interpolation.
     * `iops` - `0` If the EBS volume is not a provisioned IOPS image, otherwise the supported IOPS count.
     * `kms_key_arn` - Amazon Resource Name (ARN) of KMS Key, if EBS volume is encrypted.
     * `snapshot_id` - The ID of the snapshot.
+    * `throughput` - `0` If the EBS volume is not `gp3`, otherwise the provisioned throughput in MiB/s.
```

Comment:
I'm not entirely sure if `0` is shown for non-`gp3` volumes — where can I find this info?